settingsManager/SettingsManager.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, appName='default', user=None):
 		self.customSettings = {}
 
 		self.rootDir = cOS.ensureEndingSlash(
-			os.environ.get('ARK_CONFIG'))
+			os.environ.get('ARK_ROOT') + '/ark/config')
```

Comment:
resolution order
self.filename = "make this path manually"
self.rootDir = ARK_CONFIG <- c:/users/ie/config

first file: self.filename
c:/ie/ark/config/default.json <- self.getFilename(appName)

second file:
self.overrideSettings()

self.rootDir ARK_CONFIG c:/users/ie/config/default.user.json


c:/ie/ark/config/default.json <- self.getFilename(appName)




self.filename = c:/ie/ark/config/tacoMaker.json

self.overrideSettings() -> c:/users/ie/config/tacoMaker.user.json